kolibri/core/content/urls.py

```diff
@@ -16,4 +17,9 @@
         {},
         "downloadcontent",
     ),
+    url(
+        r'^viewcontent$',
```

Comment:
This seems rather longish `http://localhost:8080/viewcontent/928329382938293829382938` and defeats the purpose of having short links.

Could we use something shorter like `/vc/` ?  or `/rc/` (redirect content) ?

I'm not sure if Short URL was part of the use case; if not you can ignore this comment and the [other comment below](https://github.com/learningequality/kolibri/pull/5641/files#r291783115).

Reply:
The main intention was for a machine parseable permalink - we can definitely extend this later for more human friendly shortlinks.